egg-pumpkin.json

```diff
@@ -0,0 +1,82 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v2",
+        "update_url": null
+    },
+    "exported_at": "2024-12-27T22:08:25+01:00",
+    "name": "pumpkin",
+    "author": "unknown@unknown.com",
+    "description": "Pumpkin is a Minecraft server built entirely in Rust, offering a fast, efficient, and customizable experience. It prioritizes performance and player enjoyment while adhering to the core mechanics of the game. Requires around 2GB RAM to build, adjust docker.installer_limits.memory in your wing accordingly! Currently, you need to adjust the port in the config manually!",
+    "features": null,
+    "docker_images": {
+        "Alpine": "ghcr.io\/pterodactyl\/yolks:alpine"
+    },
+    "file_denylist": [],
+    "startup": ".\/pumpkin",
+    "config": {
+        "files": "{}",
+        "startup": "{}",
```

Comment:
This can be changed to:
```json
        "startup": "{\r\n    \"done\": \"Started Server took\"\r\n}",
````
So Pterodactyl can properly detect startup of the server.